website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,34 +94,35 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
-There are two ways to register an external service with Consul:
-1. If [`TransparentProxy`](/docs/connect/transparent-proxy) is enabled, the preferred method is to declare external endpoints in the [`destination`](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `ServiceDefaults`.
-1. You can add the service as a node in the Consul catalog.
+You may register an external service with Consul using `ServiceDefaults` if
```

Comment:
What is the expectated outcome in the UI? Grerater than 0 amount of linked services in terminating gateway's UI?